e2e/helpers/generated_order_helper.js

```diff
@@ -140,6 +162,9 @@ module.exports = {
       case 'Emergency protection order':
         await createEmergencyProtectionOrder(I, createOrderEventPage, order, hasAllocatedJudge);
         break;
+      case 'Discharge of care order':
```

Comment:
Maybe not to address here, but I don't like create an order e2e tests, it should be relatively obvious what the test is doing easy to spot why it fails, and with current implementation we iterate through all of them which doesn't seem business like-y. Thoughts?

Reply:
i agree with this, having any kind of test, even unit in a loop is bad idea